integration-tests/tests/run-workflow.js

```diff
@@ -12,11 +14,23 @@ const testRunWorkflowFn = _.flow(
   withWorkspace,
   withUserToken
 )(async ({ billingProject, page, testUrl, token, workflowName, workspaceName }) => {
+  await testRunWorkflowHelper(billingProject, page, testUrl, token, workflowName, workspaceName)
+})
+
+const testRunWorkflowWithV1WorkspaceFn = _.flow(
+  withV1Workspace,
+  withRegisteredUser
+)(async ({ billingProject, page, testUrl, token, workflowName, v1WorkspaceName }) => {
+  await testRunWorkflowHelper(billingProject, page, testUrl, token, workflowName, v1WorkspaceName, false)
+})
+
+const testRunWorkflowHelper = async (billingProject, page, testUrl, token, workflowName, workspaceName, newEntity = true) => {
```

Comment:
As in the other file, I'd prefer to see this defined above where it's invoked.

Reply:
Done